catalog/spatial/registry/registry-common/src/main/java/org/codice/ddf/registry/common/metacard/RegistryObjectMetacardType.java

```diff
@@ -51,11 +51,11 @@
 
     public static final String DATA_END_DATE = "data-end-date";
 
-    public static final String LINKS = "links";
+    public static final String LINKS = "registry.links";
 
-    public static final String REGION = "region";
+    public static final String REGION = "registry.region";
 
-    public static final String DATA_SOURCES = "input-data-sources";
+    public static final String DATA_SOURCES = "registry.input-data-sources";
 
```

Comment:
Why didn't we add the 'registry.' to the data-types, service-bindings and service-binding-types